src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -683,7 +683,10 @@ def engine_forward(
                 )
                 for prompt_logit in prompt_logits:
                     token_generator.generate(prompt_logit)
-                return numpy.array([self.tokens]), prompt_logits
+                yield numpy.array([token_generator.tokens]), prompt_logits, [
```

Comment:
Could you add quick local test in the description? want to test if yield works as expected here

Reply:
The tests are added as a separate PR #1330